### PR TITLE
add Tongjun's people file and update the authors.yml

### PR DESCRIPTION
**Pull request by TuringLee**
> Please copy/paste the title template from the following line:  
> 🗣Oral | 📰Poster - Paper Title  
> And delete this part before continue editing your PR

# AMC Seminar Sign Up

> Link to previous seminar meeting minutes comments issue:

#ISSUE_NUMBER

## 1. Oral / Poster Selection

> Select one of the following selections for Oral or Poster:
>
> 1. Oral should be prepared for _15 - 30_ minutes, and no more than **45** minutes, including Q/A.
> 1. Posters should be prepared for _5 - 10_ minutes, and no more than **15** minutes, including Q/A.

- [ ] 🗣 Oral (15 MIN)
- [x] 📰 Poster (5 MIN)

## 2. Paper Information

> Please fill the follow table and provide all the information as many as possible, for sharing with others better. (delete the rows that you did not fill)

| Paper           | Information                |
| --------------- | -------------------------- |
| Name            | PAPER_NAME                 |
| Recommendation  | ONE_LINE_RECOMMENDATION    |
| Prior Knowledge | KEYWORDS                   |
| Year            | PUBLISH_YEAR               |
| Publish         | CONFERENCE_OR_JOURNAL_NAME |
| Author          | AUTHOR1, et. al.           |
| GitHub          | GITHUB_LINK                |
| Blog            | BLOG_LINK                  |
| Arxiv           | ARXIV_LINK                 |

### 3. After Party 🍻

> Everyone will be welcome. You can join or not join as you want (not required by the AMC), and change your mind at any time.
>
> 1. After Party will be started right after all the meeting agenda had been fulfilled.
> 1. We will find a bar and drink beers and talk whatever we want. (we can expect that it will be related to meeting topics of the day)
> 1. The bill will be paid AA

- [ ] 🍻 YES! I'll definitely join the After Party!
- [x] 🤔 Maybe. I'd like to decide later.
- [ ] 🛌 No, I'm afraid not this time.

## 4. New Friend

> Whether you have a new friend comes with you to the club.

- [ ] 👥 I'll bring a new friend
- [x] 👤 There's no new friend from me

Please fill the following table for the new friend sign up(if any), or delete it.

| Referrer | Newcomer  | Bio                    |
| -------- | --------- | ---------------------- |
| @member  | @NEW_COMER_GH | ONE_SENTENCE_BIOGRAPHY |

----------------
<sub>
<i>

Artificial Intelligence & Machine Learning CLUB, 2018 - 2019 <a href="https://ai-ml.club">AI ML Club - AMC</a> <a href="https://ai-ml.club/manuals/member/">Member Manual</a>

</i>
</sub>

### CODE REVIEW

**Comment by newcomer** [comment]
Hi There,

I'm getting a lot of emails because you mention my username @newcomer in your PRs. 
Could you please stop doing?

Thanks,
Oleg

**Comment by cgpeter96** [comment]
please check https://travis-ci.com/BUPT/ai-ml.club/builds/121412585

File: docs/_data/authors.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ qhduan:
 
 TuringLee:
```
Comment (huan) [inline docs/_data/authors.yml]: Please use all lower case for your Github id.

File: docs/_people/TuringLee.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+
+layout: author
+title: "Turing Lee(李童俊)"
+header:
+  overlay_image: /assets/images/amc-banner.jpg 
+  caption: "AI-ML CLUB"
+excerpt: "Machine Reading Comprehension."
+author: TuringLee
```
Comment (huan) [inline docs/_people/TuringLee.md]: Please also use the all lowercase for your Github id.

**Comment by huan** [comment]
@TuringLee I had fixed the linting problems for you by sending commits to your repository because this PR had been stalled too long.

@BUPT/chairs If we need to help someone to fix their PR, we can push to their repo directly because we have the rights to do that.

Learn more from <https://help.github.com/en/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork>

**Pull request merged**